Closure_Project/rest_api/models.py

```diff
@@ -149,7 +161,7 @@ class CourseGroup(models.Model):
     course_type = models.CharField(max_length=20, choices=CourseType.choices)
     year_in_studies = models.IntegerField(choices=Year.choices)
     index_in_track_year = models.IntegerField()
-    required_course_count = models.IntegerField(null=True)
+    required_course_count = models.IntegerField(null=True, validators=[_validate_non_negative_number])
     required_points = models.IntegerField(null=True)
```

Comment:
Could we use the save validation for other fields that are integers, such as `points required`? Note some are nullable

Reply:
added to the other fields that aren't the year. should I check there larger than 2021?

Reply:
Nah, let's not make our future lives (in two months, when 2022 is out) too miserable for no real reason

Reply:
so approve?

Reply:
Idk as you assigned @ilanerukh to do it, but I'll take the liberty to assign myself and approve it (: